packages/@aws-cdk/aws-ecs/test/test.ecs-cluster.ts

```diff
@@ -321,7 +321,7 @@ export = {
     // WHEN
     cluster.addDefaultCloudMapNamespace({
       name: "foo.com",
-      type: ecs.NamespaceType.PUBLIC_DNS
+      type: cloudmap.NamespaceType.DNS_PRIVATE
```

Comment:
Shouldn't this be DNS_PUBLIC? Does this unit test actually pass? If so we might want to add a test to verify this.

Reply:
Yes, it passes the unit test.

Reply:
Then can you also have this check fail if the namespace type isn't what's passed in?